### ISSUE TITLE
Unhandled exception. System.NullReferenceException: .NETcore 7.0 

### ISSUE THREAD

**Issue by alialtog** [lucaslorentz/minicover#162]
Hi, 

We getting an error when we use instrument command. We using minicover 3.6.0 and dotnetcore 7.0

```
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at MiniCover.Core.Extensions.MethodDefinitionExtensions.ResolveOriginalMethod(MethodDefinition methodDefinition) in /home/vsts/work/1/s/src/MiniCover.Core/Extensions/MethodDefinitionExtensions.cs:line 61
   at MiniCover.Core.Instrumentation.MethodInstrumenter.InstrumentMethod(IInstrumentationContext context, Boolean instrumentInstructions, MethodDefinition methodDefinition, InstrumentedAssembly instrumentedAssembly) in /home/vsts/work/1/s/src/MiniCover.Core/Instrumentation/MethodInstrumenter.cs:line 44
   at MiniCover.Core.Instrumentation.TypeInstrumenter.InstrumentType(IInstrumentationContext context, TypeDefinition typeDefinition, InstrumentedAssembly instrumentedAssembly) in /home/vsts/work/1/s/src/MiniCover.Core/Instrumentation/TypeInstrumenter.cs:line 30
   at MiniCover.Core.Instrumentation.AssemblyInstrumenter.InstrumentAssemblyDefinition(IInstrumentationContext context, AssemblyDefinition assemblyDefinition) in /home/vsts/work/1/s/src/MiniCover.Core/Instrumentation/AssemblyInstrumenter.cs:line 99
   at MiniCover.Core.Instrumentation.AssemblyInstrumenter.InstrumentAssemblyFile(IInstrumentationContext context, IFileInfo assemblyFile) in /home/vsts/work/1/s/src/MiniCover.Core/Instrumentation/AssemblyInstrumenter.cs:line 51
   at MiniCover.Core.Instrumentation.Instrumenter.VisitAssemblyGroup(IInstrumentationContext context, InstrumentationResult result, IReadOnlyCollection`1 assemblyFiles) in /home/vsts/work/1/s/src/MiniCover.Core/Instrumentation/Instrumenter.cs:line 91
   at MiniCover.Core.Instrumentation.Instrumenter.Instrument(IInstrumentationContext context) in /home/vsts/work/1/s/src/MiniCover.Core/Instrumentation/Instrumenter.cs:line 58
   at MiniCover.Commands.InstrumentCommand.Execute() in /home/vsts/work/1/s/src/MiniCover/CommandLine/Commands/InstrumentCommand.cs:line 100
   at MiniCover.Program.<>c__DisplayClass0_2.<Main>b__3() in /home/vsts/work/1/s/src/MiniCover/Program.cs:line 51
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.<>c__DisplayClass79_0.<OnExecute>b__0()
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at MiniCover.Program.Main(String[] args) in /home/vsts/work/1/s/src/MiniCover/Program.cs:line 73
./.build/src/ci-minicover.sh: line 11:    54 Aborted                 (core dumped) minicover instrument --assemblies 'test/**/*.dll' --sources 'src/**/*.cs'
The command 'sh ./.build/src/ci-test.sh Release' returned a non-zero code: 134
```

**Comment by lucaslorentz** [comment]
This was probably fixed in v3.6.1

**Comment by alialtog** [comment]
It fixed. Thank you.
